draft-ietf-core-new-block.xml

```diff
@@ -1303,6 +1323,86 @@ X<--: Message loss (response)
+         |   ...    |
+         |     [[Observe triggered]]
+         |<---------+ NON 2.05 M:0xf5 T:0xc3 O:1321 ET=91 QB2:0/1/1024
+         |<---------+ NON 2.05 M:0xf6 T:0xc3 O:1321 ET=91 QB2:1/1/1024
+         |<---------+ NON 2.05 M:0xf7 T:0xc3 O:1321 ET=91 QB2:2/1/1024
+         |<---------+ NON 2.05 M:0xf8 T:0xc3 O:1321 ET=91 QB2:3/0/1024
+         |   ...    |
+
+]]></artwork>
+          </figure></t>
+
+        <t><xref target="b12non0"></xref> illustrates the same as <xref
+        target="b12non"></xref> but this time has eleven (11) payloads in
+        both directions which exceeds MAX_PAYLOADS. There is no loss
+        experienced.</t>
+
+        <t><figure anchor="b12non0"
+            title="Example of NON FETCH with Q-Block1 and Q-Block2 Options (Without Loss)">
+            <artwork><![CDATA[        CoAP        CoAP
+       Client      Server
+         |          |
+         +--------->| NON FETCH /path M:0x01 T:0xf1 RT=10 QB1:0/1/1024
+         +--------->| NON FETCH /path M:0x02 T:0xf2 RT=10 QB1:1/1/1024
+         +--------->| [[Payloads 3 - 9 not detailed]]
+         +--------->| NON FETCH /path M:0x0a T:0xfa RT=10 QB1:9/1/1024
+      [[MAX_PAYLOADS has been reached]]
+         |     [[MAX_PAYLOADS blocks receipt acknowledged by server]]
+         |<---------+ NON 2.31 M:0x80 T:0xfa
+         +--------->| NON FETCH /path M:0x0b T:0xfb RT=10 QB1:10/0/1024
+         |<---------+ NON 2.05 M:0x81 T:0xfb O:1334 ET=21 QB2:0/1/1024
+         |<---------+ NON 2.05 M:0x82 T:0xfb O:1334 ET=21 QB2:1/1/1024
+         |<---------+ [[Payloads 3 - 9 not detailed]]
+         |<---------+ NON 2.05 M:0x8a T:0xfb O:1334 ET=21 QB2:9/1/1024
+      [[MAX_PAYLOADS has been reached]]
+         |     [[MAX_PAYLOADS blocks acknowledged by client using
+         |       'Continue' Q-Block2]]
+         +--------->| NON FETCH /path M:0x0c T:0xfc QB2:10/1/1024
+         |<---------+ NON 2.05 M:0x8b T:0xfb O:1334 ET=21 QB2:10/0/1024
+         |   ...    |
+         |     [[Observe triggered]]
+         |<---------+ NON 2.05 M:0x91 T:0xfb O:1335 ET=22 QB2:0/1/1024
+         |<---------+ NON 2.05 M:0x92 T:0xfb O:1335 ET=22 QB2:1/1/1024
+         |<---------+ [[Payloads 3 - 9 not detailed]]
+         |<---------+ NON 2.05 M:0x9a T:0xfb O:1335 ET=22 QB2:9/1/1024
+      [[MAX_PAYLOADS has been reached]]
+         |     [[MAX_PAYLOADS blocks acknowledged by client using
+         |       'Continue' Q-Block2]]
+         +--------->| NON FETCH /path M:0x0d T:0xfd QB2:10/1/1024
+         |<---------+ NON 2.05 M:0x9b T:0xfb O:1335 ET=22 QB2:10/0/1024
+      [[Body has been received]]
+         |   ...    |
+
+]]></artwork>
+          </figure></t>
+
+        <t>If there is Q-Block1 loss, how this is handled is shown in <xref
+        target="B3non1"></xref> and <xref target="B3non2"></xref> with FETCH
+        substitued for PUT and 2.04 becomes a 2.05 response.</t> 
+
+        <t>If there is Q-Block2 loss, how this is handled is shown in <xref
```

Comment:
Q-Block2 loss may not parse well. Please reword following the same as suggested for Q-Block1

Reply:
Gone with later suggestion